scripts/download_artifacts.py

```diff
@@ -111,15 +112,23 @@ def download_artifact(artifact_name: str, artifact_id: str, token: str, outdir:
         "Authorization": f"token {token}",
         "X-Github-Api-Version": "2022-11-28",
     }
-    artifact_zip_name = artifact_name.replace(".log", ".zip")
+
+    if not artifact_zip_name:
+        artifact_zip_name = artifact_name.replace(".log", ".zip")
+
+
     r = requests.get(
-        f"https://api.github.com/repos/patrick-rivos/riscv-gnu-toolchain/actions/artifacts/{artifact_id}/zip",
+        f"https://api.github.com/repos/{repo}/actions/artifacts/{artifact_id}/zip",
         headers=params,
     )
     print(f"download for {artifact_zip_name}: {r.status_code}")
     download_binary = False
     with open(f"./temp/{artifact_zip_name}", "wb") as f:
         f.write(r.content)
+    
+    if repo != "patrick-rivos/riscv-gnu-toolchain":
```

Comment:
Why don't we try to extract the files in your repo?

Reply:
Downloading things across repos this way messes with the file naming scheme. It worked before specifically because of the way the logs were named. Doing this just makes things simpler imo. I'm thinking of refactoring this spaghetti later to make it more general

Reply:
Sounds good. Just wanted to make sure there was a reason.